### DISCUSSION TITLE
Unsupported hack version, please update your device

### DISCUSSION THREAD

**Discussion by Melectro1** [roleoroleo/yi-hack_ha_integration#123] [Q&A]
Hello,

I have successfully flashed my YI Dome cam with version 0.3.7 (yi-hack-v5).
When adding the camera in HA as a new device I get an error "Unsupported hack version, please update your device".

I don't understand, the device is running on the latest firmware version for today.

Please can someone give me a hint, what is missing?

**Comment by roleoroleo** [comment]
https://github.com/roleoroleo/yi-hack_ha_integration/issues/121
https://github.com/roleoroleo/yi-hack_ha_integration/issues/116

**Comment by Melectro1** [comment]
Hello

I'm using version 0.4.5 of the Yi hack integration and I still get the same error.

I have been trying with YI dome with FW 0.3.7 and also YI dome cloud 1080p with FW 0.3.7. 
![image](https://user-images.githubusercontent.com/84949550/232195529-ad6bb2d3-b21f-49b0-a3c7-b609334bd6f3.png)

Both cameras can stream live to vlc running in windows 10

I'm not getting any positive progress. 
![image](https://user-images.githubusercontent.com/84949550/232196121-f1d05dc3-a7a0-4176-95f1-c4a25e5d2bef.png)

Please help.

**Comment by roleoroleo** [comment]
As you can see above, at the moment yi-hack-v5 is work in progress.
Meanwhile, please, downgrade the integration to ~~0.4.4~~ 0.4.2.

**Comment by Melectro1** [comment]
Thank you for writing me an answer, as I'm new to Home assistant I find it hard as the integration never has been working.

I downgraded as you wrote to 0.4.4, but I still have the same problem. Do I also have to change the firmware?
![image](https://user-images.githubusercontent.com/84949550/232224052-319c341a-bd7f-4b56-9fc9-3cc3c8459c9a.png)

**Comment by Melectro1** [comment]
I downgraded it to 0.4.2 and now I could pass the error.

**Comment by Melectro1** [comment]
The stupid supervisor autoupdate is automatically updating back to the latest version 0.4.5. I tried to find a solution to stop it, but there are many post that says it can't be stopped. :-(
I hope that you will release a fix in near future. I only got 0.4.2 to work for a minute. ;-)

**Comment by roleoroleo** [comment]
Check yi-hack-v5 for new updates.
alienatedsec is testing a new version.
